[IMP] website: revamp '+new' screen and adapt the design

chgo-odoo opened the pull request:
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:




---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/odoo/pull/136274).

anso-odoo commented on addons/website/static/src/systray_items/new_content.scss:
```diff
@@ -1,80 +1,28 @@
+    .popover {
+        max-width: calc(var(--new-content-item-size) * 3 + #{$popover-body-padding-y} * 2 + #{$spacer} * 3);
+        transform: translateX(calc((var(--new-content-item-size) * 3 + #{$popover-body-padding-y} * 2 + #{$spacer} * 2) * -1));
+        top: calc(100% + #{$o-navbar-padding-v} * 2);
```
Nitpicking but with this value, the popover is not top-aligned with the dropdowns of the header. That said, I think the problem is only relevant because we are making this popover look like a dropdown. Maybe we could add an arrow pointing to "+ New" button and make it a real popover ?

chgo-odoo commented:
@anso-odoo I've addressed your feedback in https://github.com/odoo/odoo/pull/136274/commits/bccd7b11442c08dab0420a2e15cb18c33382ecec but I'm definitely not convinced.

It requires many calculations + we should remove the border radius of the popover if we want the arrow to be well positioned which might start to be too much changes to the default popover. What do you think?

chgo-odoo commented:
Hi @stefanorigano 👋

Could you have a look at this PR and its enterprise counterpart?

Many thanks !

stefanorigano commented on addons/website/static/src/systray_items/new_content.xml:
```diff
@@ -1,28 +1,26 @@
                         title="'New Page'">
-                    <i class="fa fa-file-o"/>
-                    <p>Page</p>
+                    <img src="/website/static/src/img/apps_thumbs/page.svg"/>
```
`alt=""` attributes are mandatory ;)
Check other occurrences as well

stefanorigano reviewed (changes requested):
Hello @chgo-odoo , thanks so much 👍 
As discussed, now that I see the final result on screen I think that there is still huge room for improvement.

I wrote some tech. remarks for your information, but I think that we have to review the design again.
Let's discuss and investigate new approaches... we can keep this PR open meanwhile ;)
